examples/meta/generator/targets/scala.json

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
If the json file is removed the corresponding folder binary_classifier, distance, etc in build/tests/meta/generated_results/scala are not created. And therefore the .dat files for scala for each category cannot be saved.